db/migrate/20220315135723_create_items.rb

```diff
@@ -7,8 +7,8 @@ def change
       t.string  :type, null: false
       t.string  :brand
       t.string  :model
-      t.decimal :price, precision: 10, scale: 2
-      t.string  :currency
+      t.decimal :price, null: false, precision: 10, scale: 2
+      t.string  :currency, null: false
```

Comment:
this should ne `enum` not `string` 
https://www.postgresql.org/docs/12/datatype-enum.html